From 22739bf2a63d8f3706c444aa6b5488d64deb57f6 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Fri, 2 Jan 2015 18:19:42 +0900
Subject: [PATCH] not use bitmap font by default

since non-ASCII characters are not showed properly with default configuration
and users should find it to fix: it annoys many users and maybe a bit hard for
beginners
---
 src/skins/skins_cfg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/skins/skins_cfg.c b/src/skins/skins_cfg.c
index 3dd2d81..e4bdfca 100644
--- a/src/skins/skins_cfg.c
+++ b/src/skins/skins_cfg.c
@@ -45,7 +45,7 @@ static const gchar * const skins_defaults[] = {
  /* general */
  "autoscroll_songname", "TRUE",
  "mainwin_font", "Sans Bold 9",
- "mainwin_use_bitmapfont", "TRUE",
+ "mainwin_use_bitmapfont", "FALSE",
  "playlist_font", "Sans Bold 8",
  "show_remaining_time", "FALSE",
  "twoway_scroll", "FALSE",
-- 
2.1.4

